server/modules/email/emailModule.js

```diff
@@ -25,6 +25,7 @@ const oath2Client = new OAuth2(clientId, clientSecret, redirectUrl);
 const developerOnboardingTemplatePath = path.join(__dirname, 'developer-onboarding-email.html');
 const successOnboardingTemplatePath = path.join(__dirname, 'success-onboarding-email.html');
 const itOffboardingPath = path.join(__dirname, 'it-offboarding-email.html');
+const successOffboardingTemplatePath = path.join(__dirname, 'success-offboarding-email.html');
```

Comment:
I think we have a lot of repeated codes here:
```
const developerOnboardingTemplatePath = path.join(__dirname, 'developer-onboarding-email.html');
const successOnboardingTemplatePath = path.join(__dirname, 'success-onboarding-email.html');
const itOffboardingPath = path.join(__dirname, 'it-offboarding-email.html');
const successOffboardingTemplatePath = path.join(__dirname, 'success-offboarding-email.html');
```
I suggest you create a function that can help abstract the process of getting email template path, something like:
```
const getEmailTemplatePath = (templateName) => {
  // logic here
}
```
Consequently, we can access any template with:
```
const itOffboardingPath = getEmailTemplatePath("it-offboarding-email.html")
```
👆 Just an idea, feel free to create your own implementation/namings.

Reply:
Implemented